CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 6.1.0 (...)
+
+# Changelog
+
+- Introduced new .signedReferences property of signature to help prevent signature wrapping attacks.
```

Comment:
The changelog is automatically generated from the title of the PR. Any additional notes should go in the README.